todo/src/reducers/index.js

```diff
@@ -0,0 +1,59 @@
+// ActionType Exports
+export const ADD_TO_LIST = 'ADD_TO_LIST';
+export const TOGGLE_COMPLETE = 'TOGGLE_COMPLETE';
+export const DELETE_ITEM = 'DELETE_ITEM';
+export const CLEAR_COMPLETE = 'CLEAR_COMPLETE';
+
+// Initial State 
+const initialState = {
+    todoList: [
+        {todo: 'Make Todo List', complete: false}
+    ]
+}
+
+// Reducer
+const reducer = (state = initialState, action) => {
```

Comment:
Good job on getting all of your reducer cases working. The toggle complete syntax and logic is really clean.